Description: Check tree version, not compiler version.
 Do not check the version string of the compiler embedded in the tree files
 against Gnatvsn.Gnat_Version_String.  Checking only the tree version number
 (Tree_IO.ASIS_Version_Number, in libgnatvsn) is sufficient.
Author: Ludovic Brenta <lbrenta@debian.org>
Forwarded: no

--- a/tools/tool_utils/asis_ul-source_table-processing.adb
+++ b/tools/tool_utils/asis_ul-source_table-processing.adb
@@ -400,7 +400,7 @@
       Add_Needed_Sources : Boolean := Mimic_gcc)
    is
    begin
-      Asis.Implementation.Initialize ("-k -ws -asis05 -sv");
+      Asis.Implementation.Initialize ("-k -ws -asis05");
 
       if True then -- Can we get rid of Only_Bodies????
          Process_Sources_From_Table
